.github/workflows/ci_accountability.yml

```diff
@@ -8,6 +8,18 @@ on:
   pull_request:
     branches-ignore:
       - "chore/l10n*"
+    paths:
+      - "*"
+      - ".github/**"
```

Comment:
I would specify each workflow file individually because otherwise you are going to run all workflows when you change just one of them.

Reply:
yes, the idea is to run the whole suite when something changes in the github configuration (not only the workflows). This could be better, but that folder is not going to change so frequently and I prefer to be conservative regarding this.

Reply:
@leio10's idea sounds good to me!